pkgs/tools/misc/entr/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "entr-${version}";
+  version = "3.2";
+
+  src = fetchurl {
+    url = "http://entrproject.org/code/${name}.tar.gz";
+    sha256 = "0ikigpfzyjmr8j6snwlvxzqamrjbhlv78m8w1h0h7kzczc5f1vmi";
+  };
+
+  builder = ./builder.sh;
```

Comment:
@jagajaga Can you direct me to some documentation or code that would help me understand better which *Phases I should be using? I tried implementing

configurePhase () # For ./configure

and

preInstall () # For make test

and I wasn't able to get the package to build.